c++.html.markdown

```diff
@@ -523,7 +524,7 @@ printMessage<10>();  // Prints "Learn C++ faster in only 10 minutes!"
 // _catch_ handlers.
 try {
     // Do not allocate exceptions on the heap using _new_.
-    throw std::exception("A problem occurred");
```

Comment:
Changed to use `std::runtime_error`. Also added the include for it, `#include <stdexcept>`.